docs/howtos/DataPlane Generation - DPG/09versioning.mdx

```diff
@@ -0,0 +1,207 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Versioning
+
+This doc details what emitters will generate for versioned specs
+
+## Single API Version
+
+If there is just one API version in the spec, we will generate
```

Comment:
Somehow it feels like you didn't finish your sentence?

Reply:
I had the same impression when reading this sentence.

Reply:
lmao yeah my b